### ISSUE TITLE
Error on confirm: e.cloneNode is not a function

### ISSUE THREAD

**Issue by lucasumberto** [sweetalert2/sweetalert2#1080]
Hi! im trying to do a basic remove confirm alert, but it throws an error

```
(index):121 Uncaught (in promise) TypeError: e.cloneNode is not a function
    at ie (sweetalert2.all.min.js:1)
    at n (sweetalert2.all.min.js:1)
    at e._main (sweetalert2.all.min.js:1)
    at e.value (sweetalert2.all.min.js:1)
    at e.o (sweetalert2.all.min.js:1)
    at e.n (sweetalert2.all.min.js:1)
    at new e (sweetalert2.all.min.js:1)
    at e (sweetalert2.all.min.js:1)
    at alerta.then ((index):121)
```
here is the code, on vuejs:

https://jsfiddle.net/lucasumberto/8rarbjtu/

What can i do to make it work?
Thanks for maintaining the plugin!

**Comment by lucasumberto** [comment]
It was a wrong parameter on the confirm swall
here is the Fix for the jsfiddle:
Swal({
       title: "It was deleted!",   
       type: "success",
       confirmButtonText: "Ok!",                                    
       });

**Comment by limonte** [comment]
Thanks for reporting @lucasumberto 

Properly fixed in https://github.com/sweetalert2/sweetalert2/releases/tag/v7.20.0

https://jsfiddle.net/ad3quksn/1179/

![image](https://user-images.githubusercontent.com/6059356/39945735-3a9be00e-5574-11e8-8e62-9a00d0c13fe5.png)

**Comment by limonte** [comment]
> sweetalert2.min.js:1 Uncaught TypeError: t.forEach is not a function
> at y (sweetalert2.min.js:1)
> at z (sweetalert2.min.js:1)
> at sweetalert2.min.js:1
> at e._main (sweetalert2.min.js:1)
> at e.value (sweetalert2.min.js:1)
> at e.value (sweetalert2.min.js:1)
> at e.Re (sweetalert2.min.js:1)
> at e.t (sweetalert2.min.js:1)
> at e [as constructor] (sweetalert2.min.js:1)
> at e.t (sweetalert2.min.js:1)

:tada: This issue has been resolved in version 8.17.6 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/sweetalert2)
- [GitHub release](https://github.com/sweetalert2/sweetalert2/releases/tag/v8.17.6)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
